SABnzbd.py

```diff
@@ -54,9 +54,6 @@
     sys.exit(1)
 
 import cherrypy
-if [int(n) for n in cherrypy.__version__.split('.')] < [8, 1, 2]:
```

Comment:
removing this check is not ideal. while we provide cherrypy with the source.. not everyone uses it. this test is to safeguard that outside versions have the features we require and work with the code we have. our version is just modified to reduce bugs.